rosa_planning/rosa-hcp-prereqs.adoc

```diff
@@ -0,0 +1,87 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-hcp-aws-prereqs
+[id="rosa-hcp-prereqs"]
+= AWS prerequisites for {hcp-title}
+
+toc::[]
+
+{hcp-title-first} provides a model that Red Hat hosts the control plane and uses your AWS account to deploy clusters.
```

Comment:
```suggestion
{hcp-title-first} provides a model that Red{nbsp}Hat hosts the control plane and uses your AWS account to deploy clusters.
```

Some people have said they experienced a bug where "Red" and "Hat" merge.

Reply:
Thanks, updating